viewing_party/party.py

```diff
@@ -1,23 +1,151 @@
 # ------------- WAVE 1 --------------------
 
+# I feel like there's a better way to test if title, genre, & rating are truthy 
+# but not sure how
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {"title": title, "genre": genre, "rating": rating}
+    if title and genre and rating:
+        return new_movie
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    has_watched = [movie] 
+    user_data = {"watched" : has_watched }

```

Comment:
When you inspect user_data (as it is passed in from the test, we can see that user_data is already a dict with a key called "watched" (see lines 69-71 of test_wave_01.py:
```python 
user_data = {
        "watched": []
}
```

Since that's the case, when you reassign user_data to a dictionary literal on line 16 then you overwrite anything that was previously in the dictionary ("friends", "watchlist"). In this example, we have a pretty bare bones user_data so the test does pass, but if user_data had other key/value pairs, we don't want to overwrite them and lose the data.

Since "watched" is already list, you don't need to put `movie` into a list. We can just write: 
```python
user_data{"watched"].append(movie)
return user_data
```